docs/contents/components/forms/year-picker/index.ja.mdx

```diff
@@ -28,59 +28,95 @@ import { YearPicker } from "@yamada-ui/calendar"
 
 ```tsx
 <VStack>
-  <YearPicker variant="outline" placeholder="outline" />
-  <YearPicker variant="filled" placeholder="filled" />
-  <YearPicker variant="flushed" placeholder="flushed" />
-  <YearPicker variant="unstyled" placeholder="unstyled" />
+  <For each={["outline", "filled", "flushed", "unstyled"]}>
+    {(variant, index) => (
+      <YearPicker key={index} variant={variant} placeholder={variant} />
+    )}
+  </For>
 </VStack>
 ```
 
 カレンダーのバリアントを変更する場合は、`variant`を設定します。
 
 ```tsx
 <VStack>
-  <YearPicker placeholder="solid" calendarVariant="solid" />
-  <YearPicker placeholder="subtle" calendarVariant="subtle" />
+  <For each={["solid", "subtle"]}>
```

Comment:
@kaehehehe 
It seems that using `<For />` has actually reduced readability. Why don't we try replacing `<For />` with something else?

Reply:
@koralle 
I think it's better to use the For component. Why do you think the readability is lower?